pallets/schemas/src/lib.rs

```diff
@@ -193,6 +200,13 @@ pub mod pallet {
 		}
 	}
 
+	#[pallet::hooks]
+	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+		fn on_runtime_upgrade() -> Weight {
+			migrations::migrate_schemas_with_additional_settings::<T>()
+		}
+	}
+
```

Comment:
This i havent not tested yet, but thats the plan, to do some upgrades and check if migration works, then we can add some try-runtime flags to have a test case

Reply:
try-runtime is meant to be used for testing, esp. storage migrations.  If you have a look at the [remove sudo storage PR](https://github.com/LibertyDSNP/frequency/pull/1066) it has a bunch of logging in the try-runtime, but logging is a good idea anyway in case something goes wrong.